### ISSUE TITLE
[Broken Website] sciencing.com - Spinning logo causes high CPU usage

### ISSUE THREAD

**Issue by washtubs** [darkreader/darkreader#3433]
**Website**
https://sciencing.com/

**How to Reproduce**
Simply navigate to https://sciencing.com/

**Expected behavior**
CPU should be steady

**Actual behavior**
CPU usage gets really high. It seems to be caused by that spinning logo actually. If I delete the logo in DOM inspector, CPU goes back to normal.

**Screenshots**

**System info:**
 - OS: Linux
 - Browser: Chrome
 - Darkreader Version: 4.9.16

**Comment by Gusted** [comment]
Hi @washtubs!

I had a look for myself and disabling/enabling Dark reader didn't changed CPU usage.
Could you check if this high CPU usage is still a thing when dark reader is disabled on the site?

Regards,
Gusted

**Comment by washtubs** [comment]
tl;dr: OK nevermind. 

It's weird. Previously I observed CPU usage was high only when dark mode was turned on and the logo was there, but now I see CPU is high only when dark mode is *turned off* and the log is there. Now CPU usage actually goes down when *turning on* dark mode... lol

I also just disabled the extension entirely and am seeing fairly high usage, like 20% not as much as before but still. I think that logo is just evil. Feel free to close.
